test/dotnet-new.Tests/GivenThatIWantANewCSApp.cs

```diff
@@ -52,6 +52,21 @@ public void When_dotnet_build_is_invoked_Then_project_builds_without_warnings()
             buildResult.Should().Pass();
             buildResult.Should().NotHaveStdErr();
         }
+
+        [Fact]
+        public void When_dotnet_new_is_invoked_mupliple_times()
+        {
+            var rootPath = Temp.CreateDirectory().Path;
+
+            new TestCommand("dotnet") { WorkingDirectory = rootPath }
+                .Execute("new");
+
+            var result = new TestCommand("dotnet") { WorkingDirectory = rootPath }
+                .ExecuteWithCapturedOutput("new");
+
+            result.Should().Fail();
+            result.Should().HaveStdErr();
```

Comment:
Given that the bug also had instances of dotnet new modifying the files placed by the previous invocation, could we add logic to ensure that the files weren't modified between runs in this test?